### DISCUSSION TITLE
Unable to verify student developer pack

### DISCUSSION THREAD

**Discussion by spacecodee** [community/community#50347] [GitHub Education]
Hi, It's Ronny, I've been trying to apply for "student developer pack" but each time I tried was reject, I've been using Github how student with past student developer pack and it was working well, now when I want to, I can't, someone could help me?
As well as, I've getting this message, but I have my name like my ID Card.

![Captura de pantalla 2023-03-17 152304](https://user-images.githubusercontent.com/79499732/226030788-027f1320-72a3-43ba-bf5d-e85b3d63c558.png)

![Captura de pantalla 2023-03-17 152706](https://user-images.githubusercontent.com/79499732/226031360-95bcc305-10c8-4bc7-9846-d19476a32e92.png)

**Comment by Ntshangase** [comment]
Hi @spacecodee 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied.  The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

**Reply by spacecodee** [reply]
Hi @Ntshangase thanks to make time to help me, by the was, do you mean if my username is different than my real name, this might not be processed?

**Reply by Ntshangase** [reply]
No that's not it, Can you try making your activity public then re-apply because right now you GitHub account activity is private. Let's try that and see.

**Reply by spacecodee** [reply]
I see, let me try it then.

**Reply by spacecodee** [reply]
Hey, I have tried it again and now I get it, thanks for your support. 😁

**Reply by Ntshangase** [reply]
Those are great news🎉.  

And please don't forget to " Mark as Answer " so that other people with similar issues can find this solution.

**Reply by an1key** [reply]
Hi, i have the same problem. But my account activity is public and profile is full... Could it be a problem that my name is written in different languages on the profile and on the proof?

**Reply by Ntshangase** [reply]
Hey, It's possible that could be the reason try change it and see

**Reply by spacecodee** [reply]
Hi @an1key that could be the problem, try to upload all of your documents in English, make sure you're using a .edu mail.

**Reply by neelkhot7** [reply]
I have done everything right. Also gone through the solution stated here, but still my account for educational access is not getting verified.

**Comment by Dineshsanwal4319** [comment]
Hi This side Dinesh,

I am trying to enroll for Student Developer package, but its getting rejected, even I have corrected the profile name in Github as well as also trying submitting with other Stuffs.

Can you please help me in same? Or is there any other way of submitting docs instead using web cam

![image](https://user-images.githubusercontent.com/60836273/228449590-181d7966-ff11-4521-9aaf-64a07f6df9db.png)

**Reply by chambrin** [reply]
<img width="306" alt="Capture d’écran 2023-03-29 à 10 59 38" src="https://user-images.githubusercontent.com/74612481/228482638-a81d61fd-1369-4c40-8e27-a9a33a0cd09b.png">

same problem, I do the same method as the previous years

**Reply by spacecodee** [reply]
Hey @Dineshsanwal4319 is yours documents in English? make sure you're using a .edu mail (it could be most likely to get approved), if you have and ID try to use your camera and take a picture where you show it.

**Reply by Dineshsanwal4319** [reply]
Yes the doc is in English, I have the collage ID card but it is without date. Therefore I am using the Admit card for submission. Will it be valid?

**Reply by spacecodee** [reply]
Maybe it could, but I've used my "calendar" of my university, that wroks for me, maybe you also could try it.

**Comment by smcdonald-realtor** [comment]
Not able to upload docs to verify information for Student Global Campus. When I click the camera option it just has a timer loading notification that spins with no results.!? How am I to submit the docs needed to move forward with the process?

**Comment by Auricmondal** [comment]
Hey, I have been trying to get verified for the students developer pack. My college doesn't provide us with an .edu mail. Moreover while taking a picture of my id card it doesn't provide me the option to upload multiple images as the validity date and my information is written on two different sides. Also, there is a bug while selecting the camera option it is not switching to the back camera. Please help me out. I have tried 6 times and it got rejected each time.

**Reply by iamreiyn** [reply]
Hi @Auricmondal 

You can scan both sides of the student ID and use any editor to combine both photos into one and then attempt to upload.

**Reply by Auricmondal** [reply]
I did and it still got rejected saying something like please use the camera as uploaded files are less trustworthy.

'Your application cannot be reviewed until you fix the following:
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.'

**Comment by lacuske** [comment]
Hi I also have the same issue, I included my full name and short bio in the profile. our student ID doesn't include date, so I combined my student ID and my transcript, but still get rejected as soon as I submitted the document, I made all the efforts to try it, but still doesn't get it approved. could you help to take a look? Thanks
![C5CB8984-8FF4-4860-BC1B-458FB577A89D](https://user-images.githubusercontent.com/98114642/236924880-fec31889-b05c-429c-81c2-828eca109e7c.jpeg)

**Comment by zooeyx** [comment]
I am experiencing the same issue

**Comment by ZapeeoSheikh** [comment]
Hi i am unable to re-verify my academic status. I am unable to find an option to re-verify my documents so that i can renew my developer pack. Please help me @spacecodee I have my edu mail and University ID Card but cant find an option to submit docs

**Comment by smcdonald-realtor** [comment]
same here!

**Comment by saurabhmention** [comment]
Hii , this side saurabh here
![Screenshot 2023-07-28 113448](https://github.com/community/community/assets/112382240/f69646ad-c02b-4a50-8de5-f24085f71971)
 
I am trying to enroll for Student Developer package, but its getting rejected, even I have corrected the profile name in Github as well as also trying submitting with other Stuffs.

Can you please help me in same?
